defaults/property-defaults.yaml

```diff
@@ -158,6 +166,45 @@ sources:
           value: 0.100000000
       quality_impressions_per_duration_s:
         - template default
+  social:
```

Comment:
This is failing to pass ci because the `templates/property/social.yaml` needs to be committed to add in the `social`. channel template.